api/resources/translations/messages-es.properties

```diff
@@ -298,72 +298,72 @@ admin.message.queue.scheduled = Programado
 analytics.anc.visits-during.description = Embarazos activos que han tenido...
-analytics.connection.error = Imposible recuperar datos en este momento
-analytics.reporting.change_time_unit = Cambiar la unidad de la hora
-analytics.reporting.clinics_size = Unidad de Informes
-analytics.reporting.date_range = Rango de la fecha
-analytics.reporting.district = Escoger distrito
-analytics.reporting.form = Escoger formulario
+analytics.connection.error = No se pudo obtener datos en este momento.
+analytics.reporting.change_time_unit = Cambiar la unidad de tiempo
+analytics.reporting.clinics_size = Unidad de informes
+analytics.reporting.date_range = Rango de fechas
+analytics.reporting.district = Seleccionar distrito
+analytics.reporting.form = Seleccionar formulario
 analytics.reporting.health_centers_size = SMO
-analytics.reporting.incomplete = Informes invalidos
+analytics.reporting.incomplete = Informes inválidos
 analytics.reporting.last_time_unit = Últimos {{quantity}} {{time_unit}}
-analytics.reporting.not_submitted = Informes que faltan
-analytics.reporting.reporting_rate = Tasa de notificación
+analytics.reporting.not_submitted = Informes faltantes
+analytics.reporting.reporting_rate = Análisis de los informes enviados
 analytics.reporting.reports = Informes
 analytics.target.add = Agregar objetivo
-analytics.target.aggregates = Objetivos agregados
-analytics.target.aggregates.disabled = Objetivos agregados están deshabilitados
-analytics.target.aggregates.error = Error obteniendo objetivos agregados
-analytics.target.aggregates.error.no.contact =
-analytics.target.aggregates.error.not.found = Error cargando objetivos agregados\: objetivo no encontrado.
+analytics.target.aggregates = Objetivos acumulados
+analytics.target.aggregates.disabled = Los objetivos acumulados están deshabilitados
+analytics.target.aggregates.error = Error al obtener los objetivos acumulados
+analytics.target.aggregates.error.no.contact = Error al cargar los objetivos acumulados. Su usuario no tiene un lugar asignado o no tiene acceso al lugar asignado. Hable con su administrador para corregir el problema.
+analytics.target.aggregates.error.not.found = Error al cargar el objetivo acumulado\: Objetivo no encontrado.
 analytics.target.aggregates.no.data = Sin datos
 analytics.target.aggregates.no.target.selected = Objetivo no seleccionado.
-analytics.target.aggregates.no.targets = No se encontraron objetivos agregados
+analytics.target.aggregates.no.targets = No se encontró el objetivo acumulado
 analytics.target.aggregates.ratio = {{pass}} de {{total}}
 analytics.target.aggregates.reported = Reportado
-analytics.target.aggregates.select.error = Error cargando objetivos agregados.
-analytics.target.aggregates.supervisees.meeting.goal =
-analytics.target.aggregates.total = 
+analytics.target.aggregates.select.error = Error al cargar el objetivo acumulado.
+analytics.target.aggregates.supervisees.meeting.goal = El objetivo de la reunión de los CHW
```

Comment:
What I understand from `target.aggregates.supervisees.meeting.goal` is something more like `Objetivos acumulados supervisan las metas de los CHW`

Reply:
Ah, so in English is: `CHWs meeting goal` 
Like people achieving their goals. I have changed it to: `Promotores de salud comunitaria que alcanzaron la meta`